bulbs/poll/migrations/0004_answer_answer_image.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import djbetty.fields
+
+
+class Migration(migrations.Migration):
```

Comment:
Since this hasn't gone out yet, can we delete all of these migrations and then just remigrate? I really prefer having as few migration files as possible.